### DISCUSSION TITLE
Further development

### DISCUSSION THREAD

**Discussion by peter88213** [peter88213/zim2obsidian#10] [Announcements]
With version 0.6 I get a result that is completely satisfying for me, so I have no intention of continuing to develop *zim2obsidian*.

Anyway, feel free to report bugs in the "issues" section, or pitch ideas here in the "Discussions" forum. 
If it is of general interest, I am sure someone from the large and vivid *Obsidian* community will be found to continue this project in some way.  

Have fun! 
Peter

**Comment by sojusnik** [comment]
Thanks for your effort!

**Comment by peter88213** [comment]
For orientation regarding the possible range of features, here is a [list of formatting that Obsidian supports](https://www.markdownguide.org/extended-syntax/).

**Reply by sojusnik** [reply]
See also:

https://help.obsidian.md/Editing+and+formatting/Basic+formatting+syntax
https://help.obsidian.md/Editing+and+formatting/Obsidian+Flavored+Markdown
https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/Guides/Markdown+Syntax
https://www.markdownguide.org/tools/obsidian/

**Reply by peter88213** [reply]
Perhaps it would be helpful for the future to compile a systematic specification and derive test cases from them (see the test data). Then it would also be easier to distinguish between bug reports and feature requests. Strictly speaking, a bug is when something goes wrong that has actually already been implemented. Everything else is classified as a feature enhancement. 

This system is reflected in the version number assignment: the second number, the minor version number, is increased when a new feature is added and documented. The third number, the patch level, is increased when a bug is fixed or the code is generally improved without adding anything substantially new.

**Reply by peter88213** [reply]
In the meantime, I've described the implemented features in a little more detail in the [README file](https://github.com/peter88213/zim2obsidian). If this does not work properly, it can be called a bug. But if that is not enough for you, you may submit a feature request.

**Reply by sojusnik** [reply]
> In the meantime, I've described the implemented features in a little more detail in the [README file](https://github.com/peter88213/zim2obsidian). If this does not work properly, it can be called a bug. But if that is not enough for you, you may submit a feature request.

Thanks for the clarification!
